### PR TITLE
v2 base

### PR DESCRIPTION
**Pull request by Joeysantoro**

### CODE REVIEW

File: contract-addresses/mainnetAddresses.ts [hunk 1]
```diff
@@ -554,10 +582,6 @@ const MainnetAddresses = {
   tribe: { artifactName: 'Tribe', address: '0xc7283b66Eb1EB5FB86327f08e1B5816b0720212B' },
   tribeOTCEscrow: { artifactName: 'OtcEscrow', address: '0xe2fE8041429e4bd51c40F92C6cDb699527171298' },
   tribeBalOtcEscrow: { artifactName: 'OtcEscrow', address: '0xfFdEe6b0261d70278f5A3093A375c282eF8266Db' },
-  tribeReserveStabilizer: {
```
Comment (ditchfieldcaleb) [inline contract-addresses/mainnetAddresses.ts:557]: Why remove this?

File: test/integration/tests/bondingcurve.ts [hunk 2]
```diff
@@ -47,7 +47,7 @@ describe('e2e-bondingcurve', function () {
     doLogging && console.log(`Environment loaded.`);
   });
 
-  describe('Reserve Stabilizer', async () => {
+  describe.skip('Reserve Stabilizer', async () => {
```
Comment (ditchfieldcaleb) [inline test/integration/tests/bondingcurve.ts:50]: We should unskip this (and fix any issues it has) until we actually ship the psm.

**Review by ditchfieldcaleb — CHANGES_REQUESTED** [review]
For the size of this thing, only 2 requested changes is pretty good imo.

File: contract-addresses/mainnetAddresses.ts (hunk 1 above)
Comment (Joeysantoro) [reply contract-addresses/mainnetAddresses.ts:557]: that address was never deployed. This is getting in the way of the test for some reason so I deleted

File: test/integration/tests/fei.ts [hunk 3]
```diff
@@ -56,14 +56,14 @@ describe('e2e-fei', function () {
       );
     });
 
-    /* Test disabled until restrictedPermissions is deployed.
+    /* Tests disabled until restrictedPermissions is deployed.
```
Comment (Joeysantoro) [inline test/integration/tests/fei.ts:59]: prefer .skip to comments

**Comment by Joeysantoro** [comment]
LGTM

**Review by ditchfieldcaleb — APPROVED** [review]

**Pull request merged**